### PR TITLE
feat: new dhedge

### PR DESCRIPTION
**Pull request by lelenei**
closes #

### CODE REVIEW

File: packages/icons/general/index.ts [hunk 1]
```diff
@@ -75,3 +75,4 @@ export * from './GearSettings'
 export * from './NFTLink'
 export * from './NFTSelected'
 export * from './Reward'
+export * from './Unknow'
```
Comment (guanbinrui) [inline packages/icons/general/index.ts:78]:
```suggestion
export * from './Unknown'
```

File: packages/mask/src/plugins/dHEDGE/UI/InvestDialog.tsx [hunk 2]
```diff
@@ -153,13 +151,7 @@ export function InvestDialog() {
...
                       `I just invested ${formatBalance(amount, token.decimals)} ${cashTag}${token.symbol} in ${
                           pool?.name
-                      }. ${
-                          isTwitter(activatedSocialNetworkUI) || isFacebook(activatedSocialNetworkUI)
-                              ? `Follow @${
-                                    isTwitter(activatedSocialNetworkUI) ? t('twitter_account') : t('facebook_account')
-                                } (mask.io) to invest dHEDGE pools.`
-                              : ''
-                      }`,
+                      }. Follow @realMaskNetwork (mask.io) to invest dHEDGE pools.`,
```
Comment (guanbinrui) [inline packages/mask/src/plugins/dHEDGE/UI/InvestDialog.tsx:154]: Why this change?

File: packages/mask/src/plugins/dHEDGE/UI/PoolViewDesk.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    const blockie = useAvatar(pool?.managerAddress ?? '')
+    const valueManaged = formatAmountPostfix(formatBalance(pool?.totalValue ?? 0, DIGIT_LENGTH))
+    const lifeTimeReturn = new BigNumber(formatBalance(pool?.performance ?? 0, DIGIT_LENGTH))
+        .minus(1)
+        .multipliedBy(100)
+        .toFixed(2)
+    const riskFactor = pool && pool?.riskFactor !== -1 ? pool?.riskFactor : '-'
+    const reward = useRewards()
+
+    //#region the invest dialog
```
Comment (guanbinrui) [inline packages/mask/src/plugins/dHEDGE/UI/PoolViewDesk.tsx:92]: If you use `#region`, you should also provide `#endregion`, which tells where it should end.

File: packages/mask/src/plugins/dHEDGE/hooks/useReward.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { useAsync } from 'react-use'
+import { API_URL } from '../constants'
+import { PluginDHedgeRPC } from '../messages'
+
+export function useRewards() {
+    return useAsync(async () => {
```
Comment (guanbinrui) [inline packages/mask/src/plugins/dHEDGE/hooks/useReward.ts:6]: You should render UI for the loading status.

**Comment by lelenei** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
Some UI issues for plugin card:
1. No tips pop out from information icon;
2. Risk Factor and Lifetime Return should be red and green, Return should have +/- sign;
3. Invest/Exit button should be black or follow the Twitter theme system;
4. The person that managed by should be show under the pool name;
5. The color of chart is different from design;
6.There is new design for invest page, please check in Figma;
截屏2021-11-16 下午5 23 56

Figma:
截屏2021-11-16 下午5 43 19
截屏2021-11-16 下午5 44 25

Other bugs:
1. The token list that allow to invest is different from app;(old issue)
截屏2021-11-16 下午5 37 23
2. My invest tab is necessary to show in plugin card;
3. Swap limit is necessary to show in swap page;
